k8s/prod/common/idam/idam-web-public.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-web-public
+  namespace: idam
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: idam-web-public
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-web-public
+    version: 0.2.3
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-public:prod-1ebdebd1
+      ingressHost: hmcts-access.service.gov.uk
+      aadIdentityName: idam
+      environment:
+        STRATEGIC_SERVICE_URL: https://idam-api.platform.hmcts.net
```

Comment:
Can we use the `.internal` URLs? I don't feel comfortable having the live Prod URLs there from day 0.

Reply:
Platform URLs restored.